### PR TITLE
337 improve logs

### PR DESCRIPTION
**Pull request by IagoLast**
#337 
---

- Add a favicon to remove 404 error in test console
- Use exquisite to log `error`, `pageerror` and `requestfailed` events

### CODE REVIEW

File: test/common/util.js [hunk 1]
```diff
@@ -119,6 +124,12 @@ function handleBrowserConsole(consoleMessage) {
     }
 }
 
+function _onRequestFailed(request) {
+    if (request.failure()) {
+        console.error(`${request.url()} --> ${request.failure().errorText}`);
```
Comment (davidmanzanares) [inline test/common/util.js]: I don't have any idea about the API, but, is it ok to call `request.failure()` twice?

Comment (IagoLast) [reply test/common/util.js]: I think so https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#requestfailure

Comment (IagoLast) [reply test/common/util.js]: first time check if null, then get the text

Comment (davidmanzanares) [reply test/common/util.js]: :+1:

**Pull request merged**